### PR TITLE
chore: upgrade sidecar image versions

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
chore: upgrade sidecar image versions

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
chore: upgrade sidecar image versions
```

### CODE REVIEW

File: charts/README.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ The following table lists the configurable parameters of the latest SMB CSI Driv
 | `image.smb.tag`                                   | csi-driver-smb docker image tag                            | `canary`                                                            |
 | `image.smb.pullPolicy`                            | csi-driver-smb image pull policy                           | `IfNotPresent`                                                      |
 | `image.csiProvisioner.repository`                 | csi-provisioner docker image                               | `registry.k8s.io/sig-storage/csi-provisioner`              |
-| `image.csiProvisioner.tag`                        | csi-provisioner docker image tag                           | `v3.1.0`                                                            |
+| `image.csiProvisioner.tag`                        | csi-provisioner docker image tag                           | `v3.1.1`                                                            |
```
Comment (humblec) [inline charts/README.md:45]: lets use `3.2.0` ?
https://github.com/kubernetes-csi/external-provisioner/blob/v3.2.0/CHANGELOG/CHANGELOG-3.2.md

Comment (andyzhangx) [reply charts/README.md]: fixed, thanks

**Pull request merged**